src/contrib/graphbuilderadapter.h

```diff
@@ -50,8 +54,8 @@ class GraphBuilderAdapter : public EventHandler {
   struct ContainerFrame {
     ContainerFrame(void* pSequence)
         : pContainer(pSequence), pPrevKeyNode(&sequenceMarker) {}
-    ContainerFrame(void* pMap, void* pPrevKeyNode)
-        : pContainer(pMap), pPrevKeyNode(pPrevKeyNode) {}
+    ContainerFrame(void* pMap, void* PrevKeyNode)
```

Comment:
Typo? Looks like you lost the 'p' prefix.

Reply:
I did that to get rid of the -Wshadow message which I admit is strange since this isn't really a shadowing situation.

Reply:
I see. Then just update the local variable to something else that doesn't start with a capital letter.